mizar/common/rpc.py

```diff
@@ -329,14 +329,13 @@ def update_network_policy_ingress(self, cidr_networkpolicy_list):
                 "cidr_type": cidr_networkpolicy.get_cidr_type_int(),
                 "bit_value": str(cidr_networkpolicy.policy_bit_value),
             }
-            item_len = count_characters(conf)
-            counter += item_len
+            item_len = len(json.dumps(conf)) + 1
+            counter = len(json.dumps(conf_list)) + item_len
             if (counter < CONSTANTS.MAX_CLI_CHAR_LENGTH):
                 conf_list.append(conf)
             if (counter + item_len > CONSTANTS.MAX_CLI_CHAR_LENGTH):
```

Comment:
counter seems here already with the addition of item_len; what is the consideration to add it ne more time?

Reply:
for line #336, it's basically checking if the command line size is going out of bound at the next loop. Since the size of the str(conf) is about the same size each loop, I am doing an approx. on the size of the command line if we are adding one more conf here. 
If we check only the current size and it's out of bound, it would be too late

Reply:
ok sounds reasonable